### ISSUE TITLE
TF_CONFIG environment variable

### ISSUE THREAD

**Issue by gogogwwb** [caicloud/ciao#86]
Do I need to import TF_CONFIG to run the tensorflow example?Prompt this error, is it due to the network?

![image](https://user-images.githubusercontent.com/51263129/140711815-ed828ccd-76e1-43c4-a997-764bbafddb81.png)
![image](https://user-images.githubusercontent.com/51263129/140712465-7fe311a3-b1d2-4400-86fa-a2cca038f5f8.png)

**Comment by gaocegege** [comment]
Yeah, it is related to the network. The TF_CONFIG will be set by TFJob operator.
